src/renderer/src/components/sideBar/treeFolder.vue

```diff
@@ -24,13 +24,13 @@
       />
       <span v-else class="text-overflow">{{ folder.name }}</span>
     </div>
-    <div v-if="!folder.isCollapsed" class="folder-contents">
-      <folder
-        v-for="(childFolder, index) of folder.folders"
-        :key="index + 'folder'"
+    <div v-show="!isCollapsed" class="folder-contents">
+      <Folder
+        v-for="childFolder of folder.folders"
+        :key="childFolder.id"
         :folder="childFolder"
         :depth="depth + 1"
-      ></folder>
+      ></Folder>
```

Comment:
Is there a reason why `v-show` was used over `v-if`? This means the entire tree is rendered at any-time instead of conditional rendering, which can cause lag for big folders